src/Volume.ts

```diff
@@ -102,48 +79,41 @@ interface VolumeDataObserver {
  * are tiled across a single large 2d image plane.
  * @typedef {Object} ImageInfo
  * @property {string} name Base name of image
- * @property {string} version schema version preferably in semver format.
- * @property {number} width Width of original volumetric data prior to downsampling
- * @property {number} height Height of original volumetric data prior to downsampling
- * @property {number} channels Number of channels
+ * @property {string} [version] Schema version preferably in semver format.
+ * @property {Vector2} originalSize XY size of the *original* (not downsampled) volume, in pixels
+ * @property {Vector2} atlasDims Number of rows and columns of z-slice tiles (not pixels) in the texture atlas
+ * @property {Vector3} volumeSize Size of the volume, in pixels
+ * @property {Vector3} regionSize Size of the currently loaded subregion, in pixels
+ * @property {Vector3} regionOffset Offset of the loaded subregion into the total volume, in pixels
+ * @property {Vector3} pixelSize Size of a single *original* (not downsampled) pixel, in spatial units
+ * @property {string} spatialUnit Symbol of physical spatial unit used by `pixelSize`
+ * @property {number} numChannels Number of channels
+ * @property {Array.<string>} channelNames Names of each of the channels to be rendered, in order. Unique identifier expected
+ * @property {Array.<Array.<number>>} [channelColors] Colors of each of the channels to be rendered, as an ordered list of [r, g, b] arrays
```

Comment:
What do the brackets around the variable name here do?

Reply:
That's JSDoc for "optional parameter"